src/App.js

```diff
@@ -1,53 +1,48 @@
 import React, { useState, useEffect } from "react";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import axios from "axios";
-
 import Home from "./components/Home";
 import Dashboard from "./components/user/Dashboard";
-import UserEdit from "./components/user/UserEdit";
-import UserDelete from "./components/user/UserDelete";
+import EditUser from "./components/user/EditUser";
+import DeleteUser from "./components/user/DeleteUser";
+import HelloUser from "./components/user/HelloUser";
 import Login from "./components/auth/Login";
 import Registration from "./components/auth/Registration";
 import LoginError from "./components/error/LoginError";
-import RegistrationError from "./components/error/RegistrationError";
+import DuplicatedRegistrationError from "./components/error/DuplicatedRegistrationError";
+import NewRegistrationError from "./components/error/NewRegistrationError";
 import NotFound from "./components/error/NotFound";
 import TodoList from "./components/todo/TodoList";
 import AddTodo from "./components/todo/AddTodo";
 
 export default function App() {
-  const [loggedInStatus, setLoggedInStatus] = useState(false);
+  const [isLoggedIn, setisLoggedIn] = useState(false);
```

Comment:
本当はこっちも・・・（修正するかどうかは任せます👍）
```suggestion
  const [isLoggedIn, setIsLoggedIn] = useState(false);
```